src/client/common/application/webviewPanels/webviewPanel.ts

```diff
@@ -29,8 +29,18 @@ export class WebviewPanel extends Webview implements IWebviewPanel {
 
     public async show(preserveFocus: boolean) {
         await this.loadPromise;
-        if (this.panel) {
-            this.panel.reveal(this.panel.viewColumn, preserveFocus);
+        if (!this.panel) {
+            return;
+        }
+
+        if (preserveFocus) {
```

Comment:
I read the explanation here: https://github.com/microsoft/vscode/issues/118764#issuecomment-806195383. So the change makes sense. I might not fully grasp the API, but it does seem like this functionality should maybe be inside the reveal function? Like if !perserveFocus and the panel is already active then reveal should just be a no-op?

In any case, this works, so thanks for the fix.